system/jlib/jtrace.cpp

```diff
@@ -1468,6 +1532,10 @@ Expected Configuration format:
         disabled: true                  #optional - disable OTel tracing
         alwaysCreateGlobalIds : false   #optional - should global ids always be created?
         alwaysCreateTraceIds            #optional - should trace ids always be created?
+        sampler:                        #optional - controls how traces are either suppressed or sampled
+          type:                         #"AlwaysOff" | "AlwaysOn" | "Ratio"
+          argument:                     #optional sampler type configuration value
```

Comment:
From a user's point of view argument is very generic.  ratio would be better.

You could argue this interface has been defined based on the implementation, rather than what would make sense from the user's viewpoint.  I am not sure if you want to go this way, but you could remove the type field, and just rely on a ratio (or percentage) field.  If it was 0 you would sample none, if 100 you would include all, otherwise you would create a ratio sampler.